cmake/std/atdm/ats2/environment.sh

```diff
@@ -171,10 +171,9 @@ if [[ "$ATDM_CONFIG_COMPILER" == "CUDA-10.1.243_"* ]]; then
   # CUDA Settings
   export CUDA_LAUNCH_BLOCKING=1
   export CUDA_MANAGED_FORCE_DEVICE_ALLOC=1
-  export TRILINOS_NVCC_WRAPPER_TMPDIR=/tmp/$(whoami)
-  if [[ ! -d ${TRILINOS_NVCC_WRAPPER_TMPDIR} ]] ; then
-    echo "Creating ${TRILINOS_NVCC_WRAPPER_TMPDIR} for nvcc wrapper!"
-    mkdir ${TRILINOS_NVCC_WRAPPER_TMPDIR}
+  if [[ ! -d /tmp/${USER} ]] ; then
+    echo "Creating /tmp/${USER} for nvcc wrapper!"
```

Comment:
If this is for nvcc_wrapper and nvcc_wrapper is using `"${TMPDIR:-/tmp}"/$(whoami)`, I think we need to set `"${TMPDIR:-/tmp}"/$(whoami)`. I am testing this from https://github.com/e10harvey/Trilinos/tree/topic/jsrun_bug_swat on vortex now....